### PR TITLE
Remove 100s timeout in addon_products_yast2

### PR DESCRIPTION
**Pull request by DrMullings**
Replaced waiting for installation pop-ups by multi tag assert_screen

- Related ticket: https://progress.opensuse.org/issues/25754
- Needles: https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/757
- Verification run: http://pinky.arch.suse.de/tests/465

### CODE REVIEW

File: tests/installation/addon_products_yast2.pm [hunk 1]
```diff
@@ -63,56 +64,65 @@ sub run {
...
-                send_key 'alt-o';
-            }
-            if (check_screen 'addon-installation-pop-up', 100) {                    # e.g. RT reboot to activate new kernel
-                $perform_reboot = 1;
-                send_key 'alt-o';                                                   # OK
+            send_key 'alt-o';                           # OK
+            my @needles   = qw(unsupported-package addon-installation-pop-up addon-installation-report);
+            my $starttime = time;
+            until (match_has_tag('addon-installation-report')) {
+                check_screen \@needles, 1;
```
Comment (okurz) [inline tests/installation/addon_products_yast2.pm]: Nope, no check_screen with non-zero timeout

**Comment by DrMullings** [comment]
New verification:
http://pinky.arch.suse.de/tests/480
@okurz

**Comment by okurz** [comment]
Still WIP?

**Comment by DrMullings** [comment]
No, is completed

File: tests/installation/addon_products_yast2.pm [hunk 2]
```diff
@@ -75,44 +75,49 @@ sub run {
...
+                    send_key 'alt-o';
+                }
+                if (match_has_tag('addon-installation-pop-up')) {
+                    $perform_reboot = 1;
+                    send_key 'alt-o';
+                }
+            } until (match_has_tag('addon-installation-report'));
+            wait_screen_change { send_key 'alt-f'; }    # finish
+            send_key 'alt-a';                           # accept
+            sleep 10;
```
Comment (okurz) [inline tests/installation/addon_products_yast2.pm]: I told you already in before: no sleeps or check_screen with non-zero timeout

Comment (DrMullings) [reply tests/installation/addon_products_yast2.pm]: Oops, thats a relict

**Pull request merged**